submissions/chabuChabu/database/trips.sql

```diff
@@ -1,36 +1,56 @@
-/*The following query is used to create a table for storing passanger details*/
+/*QUESTION  1*/
 use trips;
```

Comment:
Why are you deleting all the things you did on the previous task

Reply:
Oh my word, thanks  for pointing that out  will re-submit